tests/expand-manifest.jsonld

```diff
@@ -291,7 +291,8 @@
     }, {
       "@id": "#t0041",
       "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
-      "name": "@language: null resets the default language",
+      "name": "Language nullification",
```

Comment:
Is this better than the old test suite name that got dropped: `"@language: null"`?  Fine either way I suppose.

Reply:
Well I think it's clearer, but I'm happy to revert it

Reply:
I’d keep the name the same, as the purpose field provides that further information.